it/src/main/java/org/corfudb/universe/universe/vm/VmManager.java

```diff
@@ -284,16 +297,20 @@ private VirtualMachineCloneSpec createLinuxCustomization(ImmutableMap<String, St
     }
 
     private Result<VmManager, UniverseException> cloneAndPowerOn(VirtualMachine vm) {
-        VirtualMachinePowerState powerState = vm.getSummary().runtime.powerState;
-        if (powerState == VirtualMachinePowerState.poweredOn) {
-            return Result.ok(this);
-        } else {
-            return powerOn().map(taskInfo -> this);
+        try {
+            VirtualMachinePowerState powerState = vm.getSummary().runtime.powerState;
+            if (powerState == VirtualMachinePowerState.poweredOn) {
+                return Result.ok(this);
+            } else {
+                return powerOn().map(taskInfo -> this);
+            }
+        } catch (RuntimeException ex) {
+            return Result.error(new UniverseException(ex));
```

Comment:
Do you want a message for this exception?

Reply:
not needed, it's completely clear